R/enum.R

```diff
@@ -44,7 +44,11 @@ enum <- function(enumValues) {
   }
 
   # if no keys were specified, then use the values as keys themselves
-  names(myEnum) <- enumNames %||% myEnum
```

Comment:
Don't know what happened, all functions from utils.R are not availabile during package build.

Reply:
Reverted. Getting now:

```
==> devtools::document(roclets = c('rd', 'collate', 'namespace', 'vignette'))

i Updating ospsuite.utils documentation
i Loading ospsuite.utils
Error in enumNames %||% myEnum (from enum.R#47) : could not find function "%||%"
Calls: suppressPackageStartupMessages ... force -> source_many -> source_one -> eval -> eval -> enum
Execution halted

Exited with status 1.

```

Reply:
This is fixed now

Reply:
> Don't know what happened, all functions from utils.R are not available during package build.

@PavelBal 
One explanation could be that the default package build uses alphabetical order to run the files.
My guess is that you may need to enforce the run of utilities.R in which `%||%` is defined before the run of enum.R
This can be done by adding `#' @include utilities.R` in the documentation of this enum function
You would also get in the DESCRIPTION file a new field named `Collate` that will describe in which order your files will be run